examples/sheets/people.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  BooleanField,
+  NumberField,
+  OptionField,
+  Sheet,
+  TextField,
+} from '@flatfile/configure'
+
+import { FlatfileRecord, FlatfileRecords } from '@flatfile/hooks'
+
+// Custom fields
+import customUrlField from '../fields/url'
+
+// Custom data hooks
+import splitFieldRecordCompute from '../hooks/split-field-record-compute'
+
```

Comment:
I don't know where to put this comment.
I would rename the `imports` directory to `sample-uploads` `sample-csvs` or `test-data`

`imports` could be confusing for less experienced users because they think might think it's a code concept.

Also can we rename `employees.csv` to `people-sample.csv`.  Having the same name makes it obvious which sheet it should go with

Reply:
Re. imports direcory: Done.

Re. employees.csv: The employees.csv is actually for the `employees` sheet, not the `people` sheet so I updated it to employees-sample.csv` and changed the other one to `my-sheet-sample.csv` so the pattern is `sheet-name-sample.csv`